scripts/globals/abilities/pets/crimson_howl.lua

```diff
@@ -13,8 +14,10 @@ function onAbilityCheck(player, target, ability)
 end;
 
 function onPetAbility(target, pet, skill, summoner)
-	local duration = 30 + summoner:getMod(MOD_SUMMONING);
-	target:addStatusEffect(EFFECT_WARCRY,9,0,duration);
-	skill:setMsg(MSG_BUFF);
-	return EFFECT_WARCRY;
+    local bonusTime = utils.clamp(summoner:getSkillLevel(SUMMONING_SKILL) - 300, 0, 200);

```

Comment:
it likely worked when you tested because some other thing had already loaded magic.lua just coincidently